docs/source/deployment/databricks.md

```diff
@@ -172,7 +172,7 @@ In your newly-created notebook, put each of the below code snippets into a separ
 * Install the latest version of Kedro compatible with version `0.18.4`
 
 ```console
-%pip install "kedro[spark.SparkDataSet]~=0.18.4"
+%pip install "kedro-datasets[spark.SparkDataSet]~=1.0.0"
```

Comment:
This should be 
```suggestion
%pip install "kedro~=0.18.4" "kedro-datasets[spark.SparkDataSet]~=1.0.0"
```

I'm not sure about the position of the quotation marks though..

Reply:
Thanks, good catch. I had assumed the user would have installed Kedro, I think this is true for the changes in `visualise_charts_with_plotly.md`, though not in this case.

I made these changes in [05de014](https://github.com/kedro-org/kedro/pull/2103/commits/05de0148bf4f06b3d54cb5525c900a104cb270aa).